docs/fields/relationship.mdx

```diff
@@ -109,12 +118,15 @@ Note: If `sortOptions` is not defined, the default sorting behavior of the Relat
 
 ### Filtering relationship options
 
-Options can be dynamically limited by supplying a [query constraint](/docs/queries/overview), which will be used both for validating input and filtering available relationships in the UI.
+Options can be dynamically limited by supplying a [query constraint](/docs/queries/overview), which will be used both
+for validating input and filtering available relationships in the UI.
 
-The `filterOptions` property can either be a `Where` query directly, or a function (synchronous or asynchronous) that returns one. When using a function, it will be called with an argument object containing the following properties:
+The `filterOptions` property can either be a `Where` query directly, or a function (synchronous or asynchronous) that
+returns `null` to not filter, `false` to avoid querying the relationship or the `Where` query to use. When using a
```

Comment:
The way it's typed, it's not a function that returns null. It's just the value null - no function